ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.stories.js

```diff
@@ -22,18 +22,9 @@ export default {
     siteImage: {
       control: 'text',
     },
-    showCustomizeGasModal: {
```

Comment:
There is no "container" component used here, so all component props can be included